.vscode/settings.json

```diff
@@ -9,5 +9,6 @@
 		"dist": true // set this to false to include "dist" folder in search results
 	},
 	// Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-	"typescript.tsc.autoDetect": "off"
+	"typescript.tsc.autoDetect": "off",
+	"editor.formatOnSave": true
```

Comment:
I'm unsure on this - does this apply prettier on every save? I think that will get in the way of diff editing. IMO prettier on pre-push is a better tradeoff.

Reply:
Oops, yeah, we don't want this.